cmd/release/cmd/config.go

```diff
@@ -23,27 +25,38 @@ var configCmd = &cobra.Command{
 var genConfigSubCmd = &cobra.Command{
 	Use:   "gen",
 	Short: "generate config",
-	Long:  ``,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("Here we are!")
+	Long:  `generates a new config in the default location if it doesn't exists`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if err := config.GenConfig(); err != nil {
+			return err
+		}
+		logrus.Info("config generated")
```

Comment:
Let's remove the logger. I'm not convinced Logrus or any logger at this point is going to provide any value for this CLI utility.

Reply:
How was this resolved? Looks like logrus is still in there.

Reply:
I'm pretty sure I removed it at [29d73b2](https://github.com/rancher/ecm-distro-tools/pull/340/commits/29d73b23bb19aa1aea985a113849b331187f41fb)